sysutils/clamav-server/files/patch-etc-clamd-conf-macports.diff

```diff
@@ -33,26 +33,27 @@
 +ExtendedDetectionInfo yes
  
  # This option allows you to save a process identifier of the listening
- # daemon (main thread).
+ # daemon.
 @@ -74,7 +74,7 @@
  # It is recommended that the directory where this file is stored is
  # also owned by root to keep other users from tampering with it.
  # Default: disabled
--#PidFile /var/run/clamd.pid
+-#PidFile /run/clamav/clamd.pid
 +PidFile @PREFIX@/var/run/clamav/clamd.pid
  
  # Optional path to the global temporary directory.
  # Default: system specific (usually /tmp or /var/tmp).
-@@ -93,7 +93,7 @@
+@@ -98,8 +98,7 @@
  
  # Path to a local socket file the daemon will listen on.
  # Default: disabled (must be specified by a user)
--#LocalSocket /tmp/clamd.socket
+-#LocalSocket /run/clamav/clamd.sock
+-#LocalSocket /tmp/clamd.sock
 +LocalSocket @PREFIX@/var/run/clamav/clamd.socket
```

Comment:
Why would one put this into /opt/local ???

Reply:
Because `${prefix}/var` is the place MacPorts uses for files created by MacPorts installs. There are dozens of ports that do this, including `clam', the port on which this port is based.

A few of them: `clamav`, `dcc`, `elasticsearch`, `macports`, `mailman2`, `named`, `privoxy`, `pwcheck`, `solr`,  `squid`, `apache2`, `dbus`, `dcc`, `dovecot`, `lighttpd`, `logrotate`, `named`, `nginx`, `redis`, `rspamd`, `solr`, `squid`, `tor`.